pkg/controller/controller.go

```diff
@@ -330,6 +392,14 @@ func (c *Controller) deleteK8sResources(ns, name string) error {
 		return err
 	}
 
```

Comment:
Any thoughts on cleaning up the images of the function as part of delete process?

Reply:
Not for the moment, in this case, using docker hub (or any cloud-hosted repository) as backend for the images it is not possible to delete images. If we include a docker registry within kubeless we may want to do some maintenance and garbage collector.